src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.ImportWorkloads.targets

```diff
@@ -35,4 +35,27 @@ Copyright (c) .NET Foundation. All rights reserved.
                           NETCoreSdkVersion="$(NETCoreSdkVersion)"
                           GenerateErrorsForMissingWorkloads="true"/>
   </Target>
+
+  <!--  This target is not part of the build. Only used by dotnet workload restore command. Global property "SkipResolvePackageAssets"
+        need to be set to "true" to avoid requiring restore.-->
+  <Target Name="_FindAllReferenceWorkload" DependsOnTargets="GetSuggestedWorkloads;PrepareProjectReferences" Returns="@(_ResolvedSuggestedWorkloads)">
+    <MSBuild
+      Projects="@(_MSBuildProjectReferenceExistent)"
+      Targets="GetSuggestedWorkloads"
+      BuildInParallel="$(BuildInParallel)"
+      Properties="GenerateErrorsForMissingWorkloads=false; SkipResolvePackageAssets=true"
+      Condition="'%(_MSBuildProjectReferenceExistent.BuildReference)' == 'true' and '@(ProjectReferenceWithConfiguration)' != '' and '@(_MSBuildProjectReferenceExistent)' != ''"
```

Comment:
What's the purpose of the `'@(ProjectReferenceWithConfiguration)' != ''` clause?

Reply:
Removed